doc/design/dist_refactor/remote_executor.md

```diff
@@ -0,0 +1,121 @@
+# Design Doc: RemoteExecutor
+## Abstract
+We propose some details to implement `RemoteExecutor`:
+
+- Each `Partitioned IR(intermediate representation)` has unique `IRID`.
+- Each `Partitioned IR` with its relationship with others and its resource need are stored into etcd.
+- Each `PaddlePaddle Runtime` runs `Partitioned IR` got from etcd by `IRID`. 
+  - So the executed `Partitioned IR` can communicate with each other by `IRID` even if some of them restart.
+
+## Architect graph
+<div style="align: center">
+<img src="src/remote_executor2.png" width="700" align=center/>
```

Comment:
I think it makes more sense for the transpiler running in the cloud, because the transpiler should know the resource (e.g., how many trainers running) information (e.g., with autoscaling, the number of trainers running changes).